src/components/Tabs/Tabs.js

```diff
@@ -18,7 +18,7 @@ import getStyles from "./styles";
 export default class Tabs extends Component {
 	constructor(props) {
 		super(props);
-		const { style, initialSelectedIndex, ...childProps } = props; // eslint-disable-line no-unused-vars
+		const { style, initialSelectedIndex, onChange, ...childProps } = props; // eslint-disable-line no-unused-vars
```

Comment:
go ahead and remove the `label` prop as well since it does not been to be a property of the top level `div`